pkg/api/scylla/validation/scyllaoperatorconfig.go

```diff
@@ -63,6 +64,15 @@ func ValidateScyllaOperatorConfigSpec(spec *scyllav1alpha1.ScyllaOperatorConfigS
 		allErrs = append(allErrs, ValidateSemanticVersion(*spec.UnsupportedPrometheusVersionOverride, fldPath.Child("unsupportedPrometheusVersionOverride"))...)
 	}
 
+	if spec.ConfiguredClusterDomain != nil {
+		if len(*spec.ConfiguredClusterDomain) == 0 {
+			allErrs = append(allErrs, field.Required(fldPath.Child("configuredClusterDomain"), "can't be empty"))
+		}
+		for _, msg := range apimachineryvalidationutils.IsValidLabelValue(*spec.ConfiguredClusterDomain) {
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("configuredClusterDomain"), *spec.ConfiguredClusterDomain, msg))
+		}
```

Comment:
I think it should also check `IsFullyQualifiedDomainName`

Reply:
FullyQualifiedName requires 3 segments between dots which is not what we want. Qualified name regexp is part of label value regexp so it's validated already.

Reply:
done